R/tabyl.R

```diff
@@ -277,7 +288,14 @@ tabyl_3way <- function(dat, var1, var2, var3, show_na = TRUE, show_missing_level
     dat[[2]] <- as.factor(dat[[2]])
   }
 
-  result <- split(dat, dat[[rlang::quo_name(var3)]]) %>%
+  result <- split(dat, dat[[rlang::quo_name(var3)]])
+  # split() drops attributes, so we manually add back the 1st variable attribute.
```

Comment:
```suggestion
  # split() drops attributes, so we manually add back the label attributes.
```
I was confused by "1st" here, since this adds back two labels -- or is 1st correct and I'm missing something?

Reply:
Thanks for the review. I meant initial label attribute. Your review comment captures it correctly!